src/pipx/commands/common.py

```diff
@@ -11,16 +11,36 @@
 
 import userpath  # type: ignore
 
-from pipx import constants
+from pipx import constants, emojies
 from pipx.colors import bold, red
 from pipx.emojies import hazard, stars
 from pipx.package_specifier import parse_specifier_for_install, valid_pypi_name
 from pipx.pipx_metadata_file import PackageInfo
 from pipx.util import WINDOWS, PipxError, mkdir, rmdir
-from pipx.venv import Venv
+from pipx.venv import Venv, VenvContainer
 
 
-def expose_apps_globally(
+def expose_package_globally(
```

Comment:
This function was extracted from `install` and `upgrade`.

Reply:
You probably also need to look for injected packages in the same venv that have been injected with `--include-apps` and/or `--include-deps`.